csc/cmd/controller_list_snapshots.go

```diff
@@ -75,6 +79,18 @@ func init() {
 		"",
 		"The starting token used to retrieve paged data")
 
+	listSnapshotsCmd.Flags().StringVar(
+		&listSnapshots.sourceVolumeId,
+		"source-volume-id",
+		"",
+		"The starting token used to retrieve source-volume id")
+
+	listSnapshotsCmd.Flags().StringVar(
+		&listSnapshots.SnapshotId,
+		"snapshot-id",
+		"",
+		"The starting token used to retrieve snapshot id")
```

Comment:
Same thing here. Looks a copy and past issue. This field is described as:

>  // Identity information for a specific snapshot. This field is
  // OPTIONAL. It can be used to list only a specific snapshot.
  // ListSnapshots will return with current snapshot information
  // and will not block if the snapshot is being processed after
  // it is cut.

So I think a better description would be "ID of the snapshot to list"

Reply:
Done